packages/redux-routine/src/is-action.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External imports
+ */
+import { isPlainObject, isString } from 'lodash';
+
+/**
+ * Returns true if the given object quacks like an action.
+ *
+ * @param {*} object Object to test
+ *
+ * @return {boolean}  Whether object is an action.
+ */
+export function isAction( object ) {
+	return isPlainObject( object ) && isString( object.type );
+}
+
+/**
+ * Returns true if the given object quacks like an action and has a specific
+ * action type
+ *
+ * @param {*} object Object to test
```

Comment:
Nit: Should align the doc tag fragments

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

```suggestion
 * @param {*}      object       Object to test
```